crates/bitwarden-auth/src/identity/models/webauthn_prf_user_decryption_option.rs

```diff
@@ -0,0 +1,48 @@
+use bitwarden_crypto::EncString;
+use serde::{Deserialize, Serialize};
+
+use crate::identity::api::response::WebAuthnPrfUserDecryptionOptionApiResponse;
+
+/// SDK domain model for WebAuthn PRF user decryption option.
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
+#[cfg_attr(
+    feature = "wasm",
+    derive(tsify::Tsify),
+    tsify(into_wasm_abi, from_wasm_abi)
+)]
+pub struct WebAuthnPrfUserDecryptionOption {
```

Comment:
Is there opportunity to just re-use https://github.com/bitwarden/sdk-internal/blob/0794e100645d97d0359551939673b2d09db2204d/crates/bitwarden-crypto/src/keys/rotateable_key_set.rs#L27? I see that here the "userkeyencryptedpublickey" would be missing, but it seems otherwise this is duplicated?

Reply:
I was basing code on the existing clients code which doesn't use that yet. Could you help me understand how to properly map the properties into the `RotateableKeySet`?  Is it ok for the `userkeyencryptedpublickey` to be missing?

Reply:
Hmm, the RotateableKeySet is not really meant to be split up like this. While we could adjust it in the SDK, I do think the better solution is to just sync the missing field down from the server, because then we can completely opaquely view this cryptographic object. Ideally, auth would not even own the individual fields here but a single field containing the entire object, similar to what we do for masterpasswordunlock/auth data, account cryptographic state, etc.

Otherwise we are carrying around two separate implementations of essentially the same object, one that for some reason can only be used to decrypt the downstream with the upstream, and one that can also be used for rotation.

Maybe make this a follow-up / it needs discussion.